event.go

```diff
@@ -63,16 +63,14 @@ func NewEventFromString(message string) *Event {
 // NewEvent builds an Event from a slice of data
 func NewEvent(data []byte) *Event {
 	return &Event{
-		Data:       data,
-		Properties: make(map[string]string),
```

Comment:
These should only be created when necessary, it will decrease the payload size in AMQP when there are no Properties